### ISSUE TITLE
Consider removing per-release JSON files

### ISSUE THREAD

**Issue by chriskuehl** [chriskuehl/dumb-pypi#125]
I added support for the per-release JSON API (e.g. `/pypi/requests/1.0.0/json`) in the hopes that tools like Poetry could use it to quickly fetch dependencies during dependency resolution without needing to download the entire wheel, but they never supported this for internal registries anyway (https://github.com/python-poetry/poetry/issues/5337).

I later added the `--no-per-release-json` argument to allow disabling this since it introduces a huge number of new files per release which slows down registry generation a lot.

I think we should probably just remove that feature entirely (or at the very least disable it by default), especially with PEP658 now adopted which I'm going to add support for soon. It's a better way to achieve the same goal and will actually be supported by tools going forward.

**Comment by asottile** [comment]
yeah we delete all these files too so I'm fine to just kill the feature entirely
